static/src/javascripts/projects/common/views/acquisitions-epic-slice.html

```diff
@@ -0,0 +1,22 @@
+<div class="contributions__epic-slice-wrapper">
```

Comment:
I know it's not strictly part of this PR but can we change this from contributions to acquisitions? The epic has both CTAs: our classes should reflect this